### DISCUSSION TITLE
error when ```import {DSTest} from "ds-test/src/test.sol"; ```

### DISCUSSION THREAD

**Discussion by Lucian-Ryu** [Cyfrin/foundry-full-course-cu#390] [Q&A] [answered]
hello, im getting an error when i open ```Test.sol``` it says ```import {DSTest} from "ds-test/src/test.sol"; ``` failed to be inported as it cant find the file in that location.

but the file seams to be there.

any idea why this is happening?

thanks
![11](https://github.com/Cyfrin/foundry-full-course-f23/assets/114004261/38103d5d-ab84-4baa-858a-0fa53c4076a0)

**Comment by Lucian-Ryu** [comment, marked as answer]
ok semas like i fix it by changing ```import {DSTest} from "ds-test/src/test.sol"; ``` to ```import {DSTest} from "ds-test/test.sol"; ```

still dont really understand why that was the issue.

**Reply by ChefAharoni** [reply]
This is because 'ds-test' is located in the 'lib' folder, so you need to add 'lib' to your import.

`import {DSTest} from "lib/ds-test/src/test.sol";`

Unless you have configured a shortcut in your mappings.txt file or foundry.toml. 
If so, check what is the shortcut you have set.

**Comment by Lucian-Ryu** [comment]
I tried that and didn’t solve the problem. Somehow I made it by removing the src part in the import.

**Comment by getzelus** [comment]
If your on windows like me and installed with git bash,

the folder ds-test in lib/forge-std/lib is empty 
so download it : 
https://github.com/dapphub/ds-test/tree/e282159d5170298eb2455a6c05280ab5a73a4ef0
and replace

**Comment by k2ekimdam** [comment]
forge install foundry-rs/forge-std

Works for me.
